<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>466</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IP Device Scanner by Askay</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QHeaderView::section{
    background-color: rgb(240, 240, 240);
	border: 1px solid rgb(210, 210, 210);
    color: #000;

}

QScrollBar::horizontal{
	height: 15px;
}
QScrollBar::vertical{
	width: 15px;
}

QSpinBox::up-button,QSpinBox::down-button{
	height:10px;
	width: 20px;
}

QSpinBox{
	border: 1px solid gray;
	border-radius:3px;
}


QProgressBar:horizontal{
	text-align: center;
}

QProgressBar::chunk:horizontal {
	background-color: #ff6600;
	width: 10px;
	border-radius: 2px;
	margin-right: 2px;
}

QHeaderView::section:hover{
    background-color: rgb(200, 200, 200);
}

QHeaderView::section:checked
{
    background-color: rgb(225, 225, 225);
    color: #000;

}

QPushButton{
	border-radius: 5px;
	background-color: #ff6600 ;
	color: white;
	border: 1px solid rgb(208, 83, 0);
}

QPushButton#startBtn{
	border-width: 4px;
}

QPushButton:hover{
	background-color: rgb(229, 92, 0);
}
QPushButton:pressed{
	background-color:rgb(208, 83, 0);
	border-color:rgb(255, 136, 0);
}

QGroupBox{
	color:#ff6600;
}
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>139</y>
      <width>804</width>
      <height>301</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="logTab">
     <attribute name="title">
      <string>Realtime Log</string>
     </attribute>
     <widget class="QTextBrowser" name="logField">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>33</y>
        <width>793</width>
        <height>235</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>793</width>
        <height>29</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QCheckBox" name="logCheckBox">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>4</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string> Log</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="exportLogBtn">
       <property name="geometry">
        <rect>
         <x>583</x>
         <y>4</y>
         <width>100</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Export TXT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clearLogBtn">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>4</y>
         <width>100</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Log</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="autoScrollCheckBox">
       <property name="geometry">
        <rect>
         <x>74</x>
         <y>4</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto Scroll</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="resultTab">
     <attribute name="title">
      <string>Successful Results</string>
     </attribute>
     <widget class="QTableWidget" name="resultTable">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>33</y>
        <width>793</width>
        <height>251</height>
       </rect>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>IP Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Port</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time (ms)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Authorization</string>
       </property>
      </column>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>793</width>
        <height>29</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QCheckBox" name="followline">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>4</y>
         <width>120</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Follow last line</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="clearResultsBtn">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>4</y>
         <width>100</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Results</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exportResultsBtn">
       <property name="geometry">
        <rect>
         <x>583</x>
         <y>4</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Export CSV</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>442</y>
      <width>801</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>3</y>
      <width>263</width>
      <height>153</height>
     </rect>
    </property>
    <property name="title">
     <string>Credentials</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Logins:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Passwords:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eraseLoginsBtn">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="passwordsInput">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>44</y>
       <width>120</width>
       <height>99</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="addLoginsBtn">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="loginsInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>44</y>
       <width>118</width>
       <height>99</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="erasePasswordsBtn">
     <property name="geometry">
      <rect>
       <x>228</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPasswordsBtn">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>3</y>
      <width>111</width>
      <height>129</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>76</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeout (ms):</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeoutInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>96</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>20000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="maxThreadsInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>41</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Max threads:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>3</y>
      <width>281</width>
      <height>153</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPlainTextEdit" name="portsInput">
     <property name="geometry">
      <rect>
       <x>187</x>
       <y>44</y>
       <width>85</width>
       <height>99</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="eraseIpsBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addIpsBtn">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="erasePortsBtn">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>187</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ports:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>IP ranges:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="ipRangesInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>44</y>
       <width>170</width>
       <height>99</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="addPortsBtn">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>115</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;Copyright © Askay. 2021&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
